contentcuration/contentcuration/models.py

```diff
@@ -1381,102 +1381,232 @@ def get_nodes_with_title(cls, title, limit_to_children_of=None):
+            authors=SQArrayAgg(
+                resources.distinct("author").order_by("author"), field="author"
+            ),
+            aggregators=SQArrayAgg(
+                resources.distinct("aggregator").order_by("aggregator"),
+                field="aggregator",
+            ),
+            providers=SQArrayAgg(
+                resources.distinct("provider").order_by("provider"), field="provider"
+            ),
+            languages=SQRelatedArrayAgg(
+                descendants.exclude(language=None)
+                .distinct("language__native_name")
+                .order_by(),
+                field="language__native_name",
+                fieldname="native_name",
+            ),
+            accessible_languages=SQRelatedArrayAgg(
+                accessible_languages_query,
+                field="language__native_name",
+                fieldname="native_name",
+            ),
+            licenses=SQRelatedArrayAgg(
+                resources.exclude(license=None)
+                .distinct("license__license_name")
+                .order_by("license__license_name"),
+                field="license__license_name",
+                fieldname="license_name",
+            ),
+            kind_count=RawSQL(
+                "SELECT json_agg(row_to_json (x)) FROM ({}) as x".format(
+                    kind_count_query
+                ),
+                (),
+            ),
+            tags_list=RawSQL(
+                "SELECT json_agg(row_to_json (x)) FROM ({}) as x".format(tags_query), ()
+            ),
+            coach_content=SQCount(
+                resources.filter(role_visibility=roles.COACH), field="id"
+            ),
+            exercises=SQCount(
+                resources.filter(kind_id=content_kinds.EXERCISE), field="id"
+            ),
+        )
 
         # Get sample pathway by getting longest path
         # Using resources.aggregate adds a lot of time, use values that have already been fetched
-        max_level = max(resources.values_list('level', flat=True).distinct() or [0])
-        deepest_node = resources.filter(level=max_level).first()
-        pathway = list(deepest_node.get_ancestors()
-                       .exclude(parent=None)
-                       .values('title', 'node_id', 'kind_id')
-                       ) if deepest_node else []
-        sample_nodes = [
-            {
-                "node_id": n.node_id,
-                "title": n.title,
-                "description": n.description,
-                "thumbnail": n.get_thumbnail(),
-                "kind": n.kind_id,
-            } for n in deepest_node.get_siblings(include_self=True)[0:4]
-        ] if deepest_node else []
+        max_level = max(
+            resources.values_list("level", flat=True).order_by().distinct() or [0]
+        )
+        m_nodes = With(
+            resources.values("id", "level", "tree_id", "lft").order_by(),
+            name="m_nodes",
+        )
+        depeest_node_record = (
```

Comment:
Typo :) `deepest_node_record` - but you were consistent with it (or your IDE was!)

Reply:
God bless the IDE!